components/shared_code/src/shared_data_model/metrics.py

```diff
@@ -609,6 +609,20 @@
         near_target="14",
         sources=["calendar"],
     ),
+    "todos": Metric(
+        name="To-do's",
+        description="The number of to-do's in source code.",
+        rationale="Code should not contain traces of unfinished work. An excess of work in progress (to-do's, fixme's) "
```

Comment:
I wouldn't call it an "excess of work in progress", that would be more fitting for other metrics. Is it not directly technical debt, or some other form of clutter?

Reply:
I'll reword.